src/app/containers/main-error-page/main-error-page.component.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, CUSTOM_ELEMENTS_SCHEMA, DebugElement, ViewChild } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { MainErrorPageComponent } from './main-error-page.component';
+
+describe('MainErrorPageComponent', () => {
+  @Component({
+    selector: `exui-app-host-dummy-component`,
```

Comment:
@waynepHMCTS - I think the backticks are meant to be single quotes instead? Not sure if it matters all that much